fastlane/spec/plugins_specs/plugin_generator_spec.rb

```diff
@@ -8,7 +8,7 @@
 
 describe Fastlane::PluginGenerator do
   describe '#generate' do
-    let(:plugin_info) { Fastlane::PluginInfo.new('tester', 'Fabricio Devtoolio', 'fabric.devtools@gmail.com', 'summary') }
+    let(:plugin_info) { Fastlane::PluginInfo.new('tester_thing', 'Fabricio Devtoolio', 'fabric.devtools@gmail.com', 'summary') }
```

Comment:
I love how the tests have hardly changed, they're all passing and the implementation was completely refactored! Great job 👍